### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by stoivo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@ Jeg lager denne for å ha backup av min code og lære git.
 
 
 Jeg er klar for nye eventyr!! :sparkles:
+ You sweety
```
Comment (sls-lnluksa) [inline README.md]: O thanks :)

File: README.md [hunk 2]
```diff
@@ -3,3 +3,4 @@ Jeg lager denne for å ha backup av min code og lære git.
 
 
 Jeg er klar for nye eventyr!! :sparkles:
```
Comment (sls-lnluksa) [inline README.md:5]: Don't understand this

**Review by sls-lnluksa — CHANGES_REQUESTED** [review]
Add some more stuff here

**Review by agvald — CHANGES_REQUESTED** [review]
Add more sparkles

**Review by sls-lnluksa — APPROVED** [review]

**Review by agvald — APPROVED** [review]

**Review by agvald — CHANGES_REQUESTED** [review]
I said "add more sparkles"

**Review by agvald — APPROVED** [review]

**Review by borisgdodo — CHANGES_REQUESTED** [review]
What is this Jeg?

**Pull request closed without merging**